spec/game_spec.rb

```diff
@@ -33,12 +40,13 @@ class FakeBoard; end
   it 'loops through game logic until the game is over' do
```

Comment:
I think this is a test for when the game is over, the game ends. It doesn't actually test the game loop. The description is a bit confusing to me.